recipes/singularity/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "2.4.2" %}
+{% set sha256 = "38ae774900d3349f14a7fed1101fd9bd0f7948160aff571fb1c49d52adefd31e" %}
+
+package:
+  name: singularity
+  version: {{ version }}
+
+source:
+  fn: singularity-{{ version }}.tar.gz
+  url: https://github.com/singularityware/singularity/releases/download/{{ version }}/singularity-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [not linux]
+
+test:
+  commands:
+    - which singularity
+    - singularity --help
+
+about:
+  home: http://singularity.lbl.gov/
+  license: BSD 3-Clause
+  license_family: BSD
+  license_file: LICENSE-LBNL.md
+  summary: "Singularity: Application containers for Linux"
+
+  description: |
+    Singularity is a container platform focused on supporting "Mobility of
+    Compute"
+
+    Mobility of Compute encapsulates the development to compute model where
+    developers can work in an environment of their choosing and creation and
+    when the developer needs additional compute resources, this environment
+    can easily be copied and executed on other platforms. Additionally as the
+    primary use case for Singularity is targeted towards computational
+    portability, many of the barriers to entry of other container solutions do
+    not apply to Singularity making it an ideal solution for users (both
+    computational and non-computational) and HPC centers.
+
+  dev_url: https://github.com/singularityware/singularity
+  doc_url: http://singularity.lbl.gov/user-guide
+
+extra:
+  recipe-maintainers:
+    - jakirkham
```

Comment:
Do you mind adding me as maintainer as well. We maintain 8000 Singularity containers and a recipe in BioConda as well, which I will deprecate.

Reply:
Of course. Happy to add you @bgruening. :)

Didn't realize you already had a copy. Do you notice anything about this build that might need adjusting?

Reply:
> We maintain 8000 Singularity containers...

Wow! Am still learning about Singularity myself. Any pointers? Any interesting containers worth looking at?

Reply:
At some point I would like to talk to you and other conda-forge people about this.
https://depot.galaxyproject.org/singularity/

These are all automatically generated out of bioconda packages. Its part of our build system, to actually test the packages without any environment available. The byproduct are Docker and Singularity containers to all packages.

Reply:
Its looks great to me, merge it :)

Reply:
Absolutely! Let's chat. Maybe we can open an issue on the webpage repo?

Would be interested to learn more about how you are building these images as well. :)

Great! Thanks for lending another pair of eyes. Also have added you to the maintainers' list just now.

Reply:
Would you mind starting an issue to talk about containerization somewhere, @bgruening? Maybe the webpage repo? This question came up a few times today in different contexts. Think it would great for more people to know about what you have done here and see whether other people would be interested in contributing to this effort.

Reply:
@jakirkham do you mean the webservice repo?

Reply:
Apologies for not being more specific. Was meaning [this repo]( https://github.com/conda-forge/conda-forge.github.io/ ).

Reply:
Done: https://github.com/conda-forge/conda-forge.github.io/issues/561 :)